dev.alextren.Spot.appdata.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Looks like this file is not being used now in the manifest, can you remove it?

Reply:
done!